pkg/blobstore/readfallback/read_fallback_blob_access.go

```diff
@@ -90,5 +91,23 @@ func (ba *readFallbackBlobAccess) FindMissing(ctx context.Context, digests diges
 	if err != nil {
 		return digest.EmptySet, util.StatusWrap(err, "Secondary")
 	}
+
+	// Replicate the blobs that are present only in the secondary backend to primary
+	if ba.replicator != nil {
+		presentOnlyInSecondary, _, _ := digest.GetDifferenceAndIntersection(missingInPrimary, missingInBoth)
+		replicateGroup, replicateCtx := errgroup.WithContext(ctx)
```

Comment:
We generally only use errgroups in case parallelism is needed. In this specific case you're using the errgroup to only launch a single goroutine, meaning there's no gain here. Could you remove the use of errgroup here? You may just call ReplicateMultiple() directly.

Reply:
Good point, let me update!